ansible-playbooks/playbooks/add-ocp-resource-group.yml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
internal playbook only?

Reply:
Yes, this playbook was added to support recreating the static rg names used to deploy an OCP cluster in a workflow job.

Reply:
I see i forgot that location variable.